examples/js/controls/OrbitControls.js

```diff
@@ -133,12 +133,7 @@ THREE.OrbitControls = function ( object, domElement ) {
 			offset.applyQuaternion( quat );
 
 			// angle from z-axis around y-axis
-
-			theta = Math.atan2( offset.x, offset.z );
-
-			// angle from y-axis
-
-			phi = Math.atan2( Math.sqrt( offset.x * offset.x + offset.z * offset.z ), offset.y );
+			spherical.fromVector3( offset );
```

Comment:
`spherical.fromVector3()` uses `acos` to calculate phi, instead. I'm not sure of the ramifications of this change. May be OK.